pkg/aws/actuator/actuator.go

```diff
@@ -434,7 +434,7 @@ func (a *AWSActuator) syncSTSSecret(awsSTSIAMRoleARN string, cloudTokenPath stri
 	if err != nil {
```

Comment:
the `sLog` above here seems like it could grow a `.WithField("error", err)` or some such as well, so the error is present in logs instead of the generic `processed secret` which currently doesn't distinguish between "hooray, success!" and "uhoh, failure!".

Reply:
@danilo-gemoli I agree with Trevor here

Reply:
Agreed, done.